dashboard/lib/policies/child_account.rb

```diff
@@ -30,7 +39,11 @@ module ComplianceState
   # parent permission before the student can start using their account.
   def self.compliant?(user)
     return true unless parent_permission_required?(user)
-    user.child_account_compliance_state == ComplianceState::PERMISSION_GRANTED
+    # CPA Part 2: unlock students created before the policy went into effect
+    # who have requested parental permission but have not yet received approval.
+    return true if ComplianceState.request_sent?(user) && user_predates_policy?(user)
```

Comment:
https://github.com/code-dot-org/code-dot-org/blob/e22716b4eb86d375914e3747618da89e358b358d/dashboard/lib/policies/child_account.rb#L49-L53

Reply:
In the [application controller](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/application_controller.rb#L354), we already don't redirect to the lockout page if `user_predates_policy?` is true. Is there a phase of CPA where we will remove that exemption from the redirect in the application_controller, but still want to allow pre-existing users to avoid the lockout if their request is pending?

Reply:
cc @daynew

Reply:
The next milestone will deal with updating the routing logic: https://codedotorg.atlassian.net/browse/P20-138

Reply:
@daynew, do you know why we use the `UTC` time zone instead of `MST`, which is Colorado's standard time?

Reply:
Not for good reason. If you want to us MST instead, feel free.